cmd/cluster/create.go

```diff
@@ -17,53 +17,53 @@ import (
 	kind "sigs.k8s.io/kind/pkg/cluster"
 )
 
-// CreateClusterFlagpole is a list of cli flags for create clusters command
-type CreateClusterFlagpole struct {
-	// ImageName is the node image used for cluster creation
+// CreateFlagpole is a list of cli flags for create clusters command
+type CreateFlagpole struct {
+	// The node image used for cluster creation
 	ImageName string
 
-	// Wait is a time duration to wait until cluster is ready
+	// Time duration to wait until cluster is ready
 	Wait time.Duration
 
-	// Retain if you keep clusters running even if error occurs
+	// Whether or not to keep clusters running even if error occurs
 	Retain bool
 
-	// Weave if to install weave cni
+	// weave if to install weave cni
 	Weave bool
 
-	// Flannel if to install flannel cni
+	// flannel if to install flannel cni
 	Flannel bool
 
-	// Calico if to install calico cni
+	// calico if to install calico cni
 	Calico bool
 
-	// Kindnet if to install kindnet default cni
+	// kindnet if to install kindnet default cni
 	Kindnet bool
 
-	// DeployTiller if to install tiller
+	// Whether or not to install tiller
 	Tiller bool
 
-	// Overlap if to create clusters with overlapping cidrs
+	// Whether or not to create clusters with overlapping cidrs
 	Overlap bool
 
-	// Debug sets log level to debug
-	Debug bool
+	// Log level to debug
+	debug bool
```

Comment:
Per question/discussion with @tpantelis on Slack, might want to make this debug flag exported (Debug), parallel to the other flags here.

Reply:
Done